### DISCUSSION TITLE
Prosba vysvetlenie

### DISCUSSION THREAD

**Discussion by akofakt2** [Institut-Zdravotnych-Analyz/covid19-data#16] [Q&A]
Dobry den, 
co presne obsahuje tento dataset?
OpenData_Slovakia_Covid_Hospital_AdmissionDischarge.csv
Dakujem 
Kundrik

**Comment by MartinHBA** [comment]
za dany den pocet prijatych (admissions) do nemocnice a pocet prepustenych (discharges) z nemocnice

**Comment by akofakt2** [comment]
S covidom, alebo vsetkych

**Comment by Elkabeielika** [comment]
Dobry den. 
Co znamenaju cisla v stlpci date po 30.11. v datasete OpenData_Slovakia_Covid_Deaths_AgeGroup_District.csv?
Dakujem.

**Reply by akofakt2** [reply]
su to datumy. Ibaze verzia excelu (sic) v ktorej udrziavaju tento dataset je stara a tak to pred pouzitim musite reformatovat.
